src/main/java/server/router/ServerRouter.java

```diff
@@ -0,0 +1,23 @@
+package main.java.server.router;
+
+import main.java.handlers.DefaultHandler;
+import main.java.handlers.Handler;
+import main.java.handlers.HandlerCollector;
+import main.java.server.Response;
+import main.java.server.request.Request;
+
+public class ServerRouter implements Router {
+
+    @Override
+    public Response route(Request request) {
+        Route route = new Route(request.getRequestMethod(), request.getRequestPath());
+        var allHandlers = HandlerCollector.ALL_HANDLERS;
```

Comment:
Not urgent right now, but this strikes me as the kind of information you'd want to inject into this namespace rather than hard coding. Going back to the whole application versus server conversation, we probably wouldn't expect the server's router to be inherently aware of all routes/handlers out of the box.